words/objects.txt

```diff
@@ -361,13 +403,16 @@ chance
 change
 channel
 chanter
+character
 chard
 charger
 chartreuse
 chatter
 chauffeur
 check
 cheek
+cheese
+cheesecake
```

Comment:
this is sometimes used to refer to suggestive pictures

Reply:
news to me!